### DISCUSSION TITLE
Translation Stopped Working On Graphene OS (Android 14)

### DISCUSSION THREAD

**Discussion by Ronn-H** [you-apps/TranslateYou#469] [General]
### Steps to reproduce

1. Open the app on last build of Graphene OS 
2. Enter a word for translation (No matter which language)
3. Click on translate or wait if automatic translate is enabled
4. Progress Bar looks active but translation does not appear

### Expected behavior

Word should be translated to the requested language

### Actual behavior

No translation appear

### Translate You version

10

### Android version

Android 14

### Other details

The application worked perfectly on the phone, but suddenly stopped working lately.
It doesn't matter which tralsation engine is configured (I personally use Reverso).

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Did you attempt clearing the app's cache?

**Comment by Ronn-H** [comment]
Yes. I have 👍
And also, uninstalled and reinstalled the app on my phone...

**Comment by ghost** [comment]
Same thing happens to me, I realized it stopped working some time ago.

It also displays a small notice informing about an error with the connection to the servers.

Fairphone OS with Android 14 and app version 10.0 as well.

**Comment by mbbsas** [comment]
Same behaviour!
Lineageos 21 
Android 14
Translate you 7.0

**Comment by GitGitro** [comment]
I see you all have privacy conscious ROMs.
I'm just assuming here, but by chance are you all using a VPN service?

I noticed with my VPN and Reverso Engine that it returns a 403 error saying that the ip is blacklisted.
For you i suggest to use mozhi and the enginge you prefer, for the devs maybe a try catch to intercept those HttpExceptions?
I've tried to implement it myself but without results :(

**Comment by itstomsh** [comment]
I can confirm that this bug also occurs on my phone. I don't use a VPN, and I'm running e.os 2.5 on Android 13. It worked fine before, but I don't know when it stopped working since I don't use app that often. I tried downgrading to versions 10 and 9, but the problem persists.

**Comment by Bnyro** [comment]
Tried all engines?

**Comment by Bnyro** [comment]
Including app restarts after switching engine and clearing the cache?

**Reply by itstomsh** [reply]
I cleared the cache and data. I have now tried all the engines, but this one doesn't work: LibreTranslate, Lingva, DeepL, Reverso, and OneRing. There is a strange exception with DeepL, which works only once after switching from a good engine. The rest I mentioned do not work at all. Additionally, when selecting LibreTranslate and Lingva, I receive an error notification: "Error communicating with the server." The others do not show that message but simply do nothing.

**Comment by upbox-org** [comment]
I had the same problem on GrapheneOS (connection error). 
For some reason, it works now after I've activated the "Exploit protection compatibility mode" in the app settings.
![Screenshot_20241224-214403(1080)x(3722)2024-12-24_21-50-38_2676](https://github.com/user-attachments/assets/b3eb6f7a-dc8d-4e87-a0fa-44103cdff9b6)
